chainer/optimizers/adam.py

```diff
@@ -68,6 +69,65 @@ def _inplace_axpby(x, a, b, y):
             x[:] = a * x + b * y
 
 
+class AdamKernel(object):
```

Comment:
"kernel" might be misleading because the word only applies in cupy implementation.